native/python/zusf_py.cpp

```diff
@@ -0,0 +1,14 @@
+#include "zusf_py.h"
+#include "../c/zusf.hpp"
+
+std::string list_uss_dir(const std::string &path)
+{
+    ZUSF ctx = {0};
+    std::string out;
+    if (zusf_list_uss_file_path(&ctx, path.c_str(), out) != 0)
+    {
+        std::string error_msg = ctx.diag.e_msg;
+        throw std::runtime_error(error_msg);
```

Comment:
We can avoid declaring a block-scoped variable by passing the string directly to the `runtime_error` constructor:
```suggestion
        throw std::runtime_error(std::string(error_msg));
```